library/src/main/java/com/instacart/library/truetime/DiskCacheClient.java

```diff
@@ -1,32 +1,45 @@
 package com.instacart.library.truetime;
 
 import android.content.Context;
-import android.content.SharedPreferences;
 import android.os.SystemClock;
 
-import static android.content.Context.MODE_PRIVATE;
+import static com.instacart.library.truetime.CacheInterface.KEY_CACHED_BOOT_TIME;
+import static com.instacart.library.truetime.CacheInterface.KEY_CACHED_DEVICE_UPTIME;
+import static com.instacart.library.truetime.CacheInterface.KEY_CACHED_SNTP_TIME;
 
 class DiskCacheClient {
 
-    private static final String KEY_CACHED_SHARED_PREFS = "com.instacart.library.truetime.shared_preferences";
-    private static final String KEY_CACHED_BOOT_TIME = "com.instacart.library.truetime.cached_boot_time";
-    private static final String KEY_CACHED_DEVICE_UPTIME = "com.instacart.library.truetime.cached_device_uptime";
-    private static final String KEY_CACHED_SNTP_TIME = "com.instacart.library.truetime.cached_sntp_time";
-
     private static final String TAG = DiskCacheClient.class.getSimpleName();
 
-    private SharedPreferences _sharedPreferences = null;
+    private CacheInterface cacheInterface = null;
```

Comment:
not my smartest decision, but for convention's sake, probably should change this to `private CacheInterface _cacheInterface`

Reply:
Good catch. I did not pay attention to the original code style. Will be updated.